src/app/hooks/useClickTrackerHandler/index.jsx

```diff
@@ -63,26 +70,18 @@ const useClickTrackerHandler = (eventTrackingData = {}) => {
           event.stopPropagation();
           event.preventDefault();
 
-          if (optimizely) {
-            const eventName = OPTIMIZELY_CONFIG.viewClickAttributeId;
+          if (sendOptimizelyEvents && optimizelyVariation) {
+            const overrideAttributes = optimizely?.user.attributes;
 
-            const overrideAttributes = {
-              ...optimizely.user.attributes,
-              [`clicked_${eventName}`]: true,
```

Comment:
May want to keep this? Very hardcoded thing though, particularly for more complex experiments.

Reply:
Have we used this in previous experiments? Now that we're tracking component views & clicks by name, maybe we don't need it anymore?